circom/client_side_proofs/oprf_delegate.circom

```diff
@@ -37,14 +38,15 @@ template OprfDelegate(MAX_DEPTH, RP_MAX_DEPTH) {
     signal input query_s;
     signal input query_r[2];
    // Credential Signature
-    signal input cred_type_id;
+    signal input issuer_schema_id; // Public
     signal input cred_pk[2]; // Public
     signal input cred_hashes[2]; // [claims_hash, associated_data_hash]
     signal input cred_genesis_issued_at;
     signal input cred_expires_at;
     signal input cred_s;
     signal input cred_r[2];
     signal input current_time_stamp; // Public
+    signal input cred_genesis_issued_at_min; // Public
```

Comment:
if the RP doesn't wish to use this, is there a way we can treat 0 as ignore? or should we translate this to the max value in the rust inputs?